lib/types/dns.go

```diff
@@ -213,33 +213,37 @@ func (c *DNSConfig) UnmarshalText(text []byte) error {
 		*c = DefaultDNSConfig()
 		return nil
 	}
-	params, err := strvals.Parse(string(text))
-	if err != nil {
-		return err
+	values := strings.Split(string(text), ",")
+	params := make(map[string]string, len(values))
+	for _, value := range values {
+		args := strings.SplitN(value, "=", 2)
+		if len(args) != 2 {
+			return fmt.Errorf("no value for key %s", value)
+		}
+		params[args[0]] = args[1]
```

Comment:
I'd rather see this as a separate helper function, but we can move it later.

There's also a similar [`lib.SplitKV()`](https://github.com/grafana/k6/blob/5c14dae129a5eb6ed309c94a194370878b7b08e5/lib/util.go#L27-L34).

Reply:
I was initially going to use https://github.com/grafana/k6/blob/master/log/tokenizer.go#L87 , but then I decided that is too much for 6 lines.

SplitKV also just makes a value empty if there is only a key, which I don't think is ... a good idea :thinking: